R/model_functions.R

```diff
@@ -93,7 +93,11 @@ fit_bayesian_model <- function(focal_vs_comp, prior_param = NULL, run_shuffle =
     V_star = V_star,
     sp_list = sp_list
   )
-  return(posterior_hyperparameters)
+
+  structure(
+    posterior_hyperparameters,
+    class = c("fe_bayes_lm", class(posterior_hyperparameters))
+  )
```

Comment:
huh, so this is all it takes to define an S3 class. Interesting 🤔

Reply:
Hahaha yup! Some folks even just `class(x) <- "newclass"`. :-)